### PR TITLE
Change slogan to “Your Secure Workspace”

### PR DESCRIPTION
**Pull request by MariusBluem**
* according to https://github.com/nextcloud/nextcloud.com/commit/d698ddba6a29f8074de9e601bcdb84c7cacf2055

Is it needed to adjust the `l10n`-files? 🤔 @MorrisJobke 

Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: tests/data/emails/new-account-email-custom.html [hunk 1]
```diff
@@ -184,4 +184,4 @@ <h1 class="text-center" style="Margin:0;Margin-bottom:10px;color:inherit;font-fa
 		<!-- prevent Gmail on iOS font size manipulation -->
 		<div style="display:none;white-space:nowrap;font:15px courier;line-height:0">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</div>
 	</body>
-</html>
```
Comment (LukasReschke) [inline tests/data/emails/new-account-email-custom.html]: I think by adding a newline here you broke the tests :-)

See https://drone.nextcloud.com/nextcloud/server/8336/2

**Review by LukasReschke — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
> Is it needed to adjust the l10n-files?

No ❗️

**Comment by nickvergessen** [comment]
I dislike this change.

**Comment by jospoortvliet** [comment]
I would certainly wait. We changed the slogan on the front page after some discussion where we couldn't come to a clear agreement so decided to try and see what ppl say ;-)

Still waiting, didn't hear any spontaneous comments yet...

**Pull request closed without merging**